internal/storagenode/logstream/executor_test.go

```diff
@@ -80,7 +80,9 @@ func TestExecutor_Closed(t *testing.T) {
 	assert.NoError(t, lse.Close())
 	assert.Equal(t, executorStateClosed, lse.esm.load())
 
-	_, err := lse.Append(context.Background(), TestNewBatchData(t, 1, 0))
+	appendTask := NewAppendTask()
+	defer appendTask.Release()
+	err := lse.AppendAsync(context.Background(), TestNewBatchData(t, 1, 0), appendTask)
```

Comment:
Is there no need to wait for the task to complete?

Reply:
No, there isn't. This invocation fails fast, so it doesn't have a result to wait for.